bake/bake.go

```diff
@@ -248,10 +248,10 @@ func (t *Target) normalize() {
 	t.Outputs = removeDupes(t.Outputs)
 }
 
-func TargetsToBuildOpt(m map[string]Target) (map[string]build.Options, error) {
+func TargetsToBuildOpt(m map[string]Target, noCache, pull bool) (map[string]build.Options, error) {
```

Comment:
@tonistiigi happy to put this in a struct (what name? `cliOpts`, `userOpts`, `extraOpts`, ...) or if you have better ideas.